cogs/moderation/moderation.py

```diff
@@ -102,8 +101,9 @@ async def _ban(self, ctx: Context, member: discord.Member, *, reason="Breaking T
             await ctx.send(embed=embed)
 
             await self.bot.db.execute(
-                '''INSERT INTO Cases(id, guildid, userid, modid, username, modname, case_type, created_at, expires_at)
-                VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)''', next(self.bot.idgen), ctx.guild.id, member.id, ctx.author.id, member, ctx.author, "ban", curtime, "never")
+                '''INSERT INTO Cases(id, guildid, userid, modid, username, modname, case_type, created_at, expires_at, case_data)
+                VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10)''', next(self.bot.idgen), ctx.guild.id, member.id,
+                ctx.author.id, member, ctx.author, "ban", curtime, "never", reason)
```

Comment:
Same as the other comment - I have a feeling it'll error while inserting `member`, `ctx.author`